### ISSUE TITLE
Unify the default notation of parameters in docstrings

### ISSUE THREAD

**Issue by tkoyama010** [pyvista/pyvista#2284]
**Describe what maintenance you would like added.**

- A clear and concise description of the maintenance problem.

In some cases, such as the following source code, the notation for default is not the standard notation for [numpydoc docstrings](https://numpydoc.readthedocs.io/en/latest/format.html#parameters).
Please check all the source codes of pyvista and make the notation consistent.

- If you are requesting maintenance for a specific line of source code, please include the permanent link for that line.

https://github.com/pyvista/pyvista/blob/bd2fe9d4a9439ad55c977a567e987752181320ff/pyvista/utilities/geometric_objects.py#L825-L826

- Feel free to include pseudocode or screenshots of the requested outcome.

```python
    depth : float, default 0.5
         Depth of the text.
```

-----
_Originally posted by @adeak in https://github.com/pyvista/pyvista/pull/2175#discussion_r815827240_

**Comment by adeak** [comment]
Thanks for tracking this @tkoyama010.

We should look at the built docs artifact in #2175; I wonder if there's a difference in the rendered result (probably not). I also wonder about the good first issue label, for the sole reason that if we want to unify this it would lead to a huge diff, which would probably be daunting for a new users and would need careful review.

**Comment by tkoyama010** [comment]
Yeah. It does not affect the rendering. It may be better to attribute this to document distortions.

**Comment by MatthewFlamm** [comment]
`numpydoc` is not dogmatic on this:

> Optional keyword parameters have default values, which are displayed as part of the function signature. They can also be detailed in the description

**Comment by tkoyama010** [comment]
My personal preference is to write simply, so it's not a high priority :)

**Comment by tkoyama010** [comment]
Seems this is already solved. Closing.

**Comment by akaszynski** [comment]
Resolved with #4155.
